strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/nuget/NugetPackageController.java

```diff
@@ -248,11 +253,13 @@ private void writePackagePart(String boundaryString,
                 return;
             }
 
-            multipartStream.readBodyData(packagePartOutputStream);
+            int contentLength = multipartStream.readBodyData(packagePartOutputStream);
+            logger.info(String.format("NuGet package content length [%s]", contentLength));
         }
         catch (MultipartStream.MalformedStreamException e)
         {
             // Seems that this is normal for Nuget push request
```

Comment:
@carlspring now it's seems that this shouldn't happen, and I need some time to resolve this behavior :)
it works file without the exception in test